store/rootmulti/store.go

```diff
@@ -12,11 +12,11 @@ import (
 
 	"github.com/cosmos/cosmos-sdk/store/cachemulti"
 	"github.com/cosmos/cosmos-sdk/store/dbadapter"
-	"github.com/cosmos/cosmos-sdk/store/errors"
```

Comment:
lol what was this?

Reply:
lol it was to avoid import cycles 🤦‍♂ but its removed now!